src/Components/NavBar/NavBar.tsx

```diff
@@ -194,8 +188,7 @@ export const NavBar: React.FC = track(
                 flex={1}
                 alignItems="center"
                 onFocus={() => {
-                  if (!isSearchDropDownImprovementsEnabled)
-                    setSearchFocused(true)
+                  setSearchFocused(true)
```

Comment:
Is this still needed? (or should it not be here as it is enabled)

Reply:
Oh, I got the boolean wrong when I was scanning this.

Reply:
Lol what was this even doing!? It was on a div that couldn't be focused.

Reply:
Lots of code for that mixed into this parent component — all that can and should be isolated.